RadixWallet/Clients/TokenPriceClient/TokenPriceClient+Live.swift

```diff
@@ -45,10 +42,12 @@ extension TokenPricesClient {
 extension TokenPricesClient.TokenPrices {
 	fileprivate init(_ tokenPricesResponse: TokensPriceResponse) {
 		let formatter = NumberFormatter()
```

Comment:
Would be nice to have a RETDecimal initializer in Sargon to handle the case of providing a Double with more than 18 decimal places.

Reply:
yeah I can fix one!